### PR TITLE
Fixed Broken Link

### PR DESCRIPTION
**Pull request by mbssaiakhil**
Fixed Broken Link in Case Study of Box

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4798)

### CODE REVIEW

File: case-studies/box.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
...
+      Another plus: Ghods liked that <a href="http://kubernetes.io/">Kubernetes</a> has a universal set of API objects like pod, service, replica set and deployment object, which created a consistent surface to build tooling against. "Even PaaS layers like <a href="https://www.openshift.com/">OpenShift</a> or <a href="http://deis.io/">Deis</a> that build on top of Kubernetes still treat those objects as first-class principles," he says. "We were excited about having these abstractions shared across the entire ecosystem, which would result in a lot more momentum than we saw in other potential solutions."<br><br>
```
Comment (chenopis) [inline case-studies/box.html]: Actually, could you please make it https, i.e.: `https`://kubernetes.io/

**Comment by mbssaiakhil** [comment]
Actually, in this document there is one more place, where http://kubernetes.io is already mentioned, should I change that also to https://kubernetes.io ?

And there are few other links such as,

http://php.net/
http://deis.io/

Should I change them also to https?

I also observed one more thing in all the case studies that, in the starting sections like *Challenge* and *Solution*, few of the links are not working. Fox example in this case study, in *Challenge* section the link to **Box** website is not clickable.

File: case-studies/box.html (hunk 1 above)
Comment (mbssaiakhil) [inline case-studies/box.html]: Actually, in this document there is one more place, where http://kubernetes.io is already mentioned, so I thought I have to use http. When I checked as well the http also works as it automatically gets redirected to https. So, should I change that also to https://kubernetes.io ?

And there are few other links such as,

http://php.net/
http://deis.io/

Should I change them also to https?

I also observed one more thing in all the case studies that, in the starting sections like Challenge and Solution, few of the links are not working. Fox example in this case study, in Challenge section the link to Box website is not clickable.

Comment (mbssaiakhil) [inline case-studies/box.html]: @chenopis, can you please kindly suggest how should I go about this?

Comment (mbssaiakhil) [inline case-studies/box.html]: @chenopis, could you please look into the above query?

Comment (radhikapc) [reply case-studies/box.html]: @mbssaiakhil go ahead and update to https://kubernetes.io

Comment (mbssaiakhil) [reply case-studies/box.html]: Thanks @radhikapc, I have amended the commit accordingly. Kindly review and merge.

**Comment by mbssaiakhil** [comment]
@chenopis / @radhikapc, I have amended my commit as required, kindly review and merge

**Pull request merged**